subsys/bluetooth/mesh/op_agg.c

```diff
@@ -58,22 +54,23 @@ int bt_mesh_op_agg_send(struct bt_mesh_model *model,
 			struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *msg,
 			const struct bt_mesh_send_cb *cb)
 {
-	uint16_t src;
 	int err;
 
 	/* Model responded so mark this message as acknowledged */
 	agg_ctx.ack = true;
 
+#ifdef CONFIG_BT_MESH_OP_AGG_CLI
```

Comment:
It is preferrable to use:
```
if (IS_ENABLED(CONFIG_FOO)) {
    do_something_with_foo
}
```
since the compiler can generate errors and warnings for `do_something_with_foo` even when CONFIG_FOO is undefined.